tests/conftest.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import pytest
+import os
+
+import launch
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def returns_returns_1(x):
+    def returns_1(y):
+        return 1
+
+    return returns_1
+
+
+def returns_returns_2(x):
+    def returns_2(y):
+        return 2
+
+    return returns_2
+
+
+ENV_PARAMS = {
+    "framework_type": "pytorch",
+    "pytorch_image_tag": "1.7.1-cuda11.0-cudnn8-runtime",
+}
+
+
+BUNDLE_PARAMS = {
+    "model_bundle_name": "test-returns-1",
+    "model": 1,
+    "load_predict_fn": returns_returns_1,
+    "env_params": ENV_PARAMS,
+    "requirements": [],
+}
+
+
+BUNDLE_PARAMS_2 = {
+    "model_bundle_name": "test-returns-2",
+    "model": 1,
+    "load_predict_fn": returns_returns_2,
+    "env_params": ENV_PARAMS,
+    "requirements": [],
+}
+
+
+@pytest.fixture()
+def client():
+    # to run this test locally, you should add the API KEY to your environment variables
+    launch_test_api_key = os.getenv("LAUNCH_TEST_API_KEY")
+    client = launch.LaunchClient(api_key=launch_test_api_key)
+
+    # these two model bundles are used in test_endpoints.py
```

Comment:
Not sure how this comment relates to the commented out `client.create_model_bundle` calls 🤔